src/client/lazy-app/icons/index.tsx

```diff
@@ -11,6 +11,46 @@ const Icon = (props: preact.JSX.HTMLAttributes) => (
   />
 );
 
+/* Cross with rounded linecaps */
+export const RoundedCrossIcon = (props: preact.JSX.HTMLAttributes) => (
+  <svg viewBox="0 0 480 480" fill="currentColor">
+    <path
+      d="M119.356 120L361 361M360.644 120L119 361"
+      stroke="currentColor"
+      stroke-width="37"
+      stroke-linecap="round"
+    />
+  </svg>
+);
+
+export const InfoIcon = (props: preact.JSX.HTMLAttributes) => (
+  <svg
+    viewBox="0 0 520 520"
+    fill="currentColor"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g clip-path="url(#a)">
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M520 0H0v520h520V0ZM230.122 77.186c-8.096 7.7587-12.144 17.3727-12.144 28.842 0 11.469 4.048 21.083 12.144 28.842s17.71 11.638 28.842 11.638c11.132 0 20.578-3.879 28.336-11.638 8.096-7.759 12.144-17.373 12.144-28.842 0-11.4693-4.048-21.0833-12.144-28.842-7.758-8.096-17.204-12.144-28.336-12.144-11.132 0-20.746 4.048-28.842 12.144Zm1.012 106.766h-5.566v199.364c0 10.795 1.856 19.903 5.566 27.324 4.048 7.084 9.108 12.987 15.18 17.71 6.41 4.385 13.325 7.59 20.746 9.614 7.759 2.024 15.518 3.036 23.276 3.036h3.036V243.154c0-20.577-6.072-35.589-18.216-45.034-11.806-9.445-26.48-14.168-44.022-14.168Z"
+        fill="currentColor"
+      />
+    </g>
+    <defs>
+      <clipPath id="a">
+        <rect width="520" height="520" rx="70" fill="currentColor" />
+      </clipPath>
+    </defs>
+  </svg>
+);
+
+export const DiamondStarIcon = (props: preact.JSX.HTMLAttributes) => (
+  <svg viewBox="0 0 48 48" fill="currentColor">
```

Comment:
```suggestion
  <svg viewBox="0 0 48 48" fill="currentColor" {...props}>
```

Otherwise the parameter is unused.